rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -99,20 +99,23 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 #
 # -=[ SQL Function Names ]=-
 #
-# This rule is a less stricter sibling of 942150.
+# This rule is a less strict sibling of 942150.
 #
-# Regexp generated from util/regexp-assemble/data/942151.data using Regexp::Assemble.
-# To rebuild the regexp:
-#   cd util/regexp-assemble
-#   ./regexp-assemble.py 942151
+# This rule has a stricter sibling 942152 that checks for SQL function names in 
```

Comment:
```suggestion
# This rule has a stricter sibling to this rule (942152) that checks for SQL function names in 
```

Reply:
Thank you for this hint. I'm not very good at finding the correct wording. So this is very welcome!